dataeng/jobs/analytics/DBTSourceFreshness.groovy

```diff
@@ -0,0 +1,59 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+
+class DBTSourceFreshness{
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("dbtsource-freshness"){
```

Comment:
Also just occurred to me this should probably be "dbt-source-freshness".

Reply:
Otherwise, LGTM

Reply:
Done